tox.ini

```diff
@@ -533,6 +533,8 @@ commands =
 
 description = check for potential packaging problems
 
+depends = {[coverage_report]depends}
```

Comment:
`packaging` environment is confused by I/O in the source tree, so this waits on unit tests to finish first (only matters if using parallel mode).